source/main/utils/Utils.cpp

```diff
@@ -37,6 +39,54 @@
 
 using namespace Ogre;
 
+void InitDiscord()
+{
+    DiscordEventHandlers handlers;
+    memset(&handlers, 0, sizeof(handlers));
+    handlers.ready = handleDiscordReady;
+    handlers.errored = handleDiscordError;
+
+    // Discord_Initialize(const char* applicationId, DiscordEventHandlers* handlers, int autoRegister, const char* optionalSteamId)
+    Discord_Initialize("492484203435393035", &handlers, 1, "1234");

```

Comment:
Where does the appID come from?

Reply:
This comes from the Rigs of Rods App in the Discord Developer portal.

Reply:
@only-a-ptr You should have been added to the team a while ago.

https://discordapp.com/developers/teams

Reply:
Oh, apparently I am, I just didn't pay attention to it 😊